Src/BootCamp.Chapter/Lesson10.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson10
+    { 
+
```

Comment:
Blank line is not needed

Reply:
I removed the whole Lesson 10 file and class